README.md

```diff
@@ -60,36 +60,32 @@ Now run Go launcher and navigate to "http://localhost:8000/quick-launch?url=" pa
 e.g."http://localhost:8000/quick-launch?url=http://localhost:7777/1_0001.json"
 ```
 
-### Deployment with [Helm](https://helm.sh/)
+### Deployment with [Concourse](https://concourse-ci.org/)
 
-To deploy this application with helm, you must have a kubernetes cluster already running and be logged into the cluster.
-
-Log in to the cluster using:
-```
-gcloud container clusters get-credentials survey-runner --region <region> --project <gcp_project_id>
-```
-
-You need to have Helm installed locally
-
-1. Install Helm with `brew install kubernetes-helm` and then run `helm init --client-only`
-
-1. Install Helm Tiller plugin for tillerless deploys `helm plugin install https://github.com/rimusz/helm-tiller`
+To deploy this application with Concourse, you must have a Kubernetes cluster already and be logged in to a Concourse instance.
 
 ---
 
-The following environment variables can be set when deploying the app.
+The following environment variables should be set when deploying the app:
 - RUNNER_URL
-- DOCKER_REGISTRY *(optional)*
-- IMAGE_TAG *(optional)*
+- PROJECT_ID
+
+The following are optional variables that can also be set if needed:
+- DOCKER_REGISTRY
+- IMAGE_TAG
+- REGION
 
 To deploy to a cluster you can run the following command
 
+```sh
+RUNNER_URL=<runner_instance_url> PROJECT_ID=<project_id> fly -t <target_concourse_instance> \
+--config ci/deploy.yaml --input eq-questionnaire-launcher-repo=.
 ```
-./k8s/deploy_app.sh
-```
-##### Example
+
+##### For Example:
  ```
-RUNNER_URL=https://example.com ./k8s/deploy_app.sh
+RUNNER_URL=https://example.com PROJECT_ID=my-project-id fly -t census-eq-ci \
```

Comment:
When I run this I get `error: unknown flag `config'`. Doesn't fly need to above version `x`?

Reply:
Runs fine for me - is your fly binary on the correct version for our concourse instance? (Concourse was recently updated)

Reply:
`execute` missing from the example - I've updated it